components/ambient-control-plane/Dockerfile.simple

```
@@ -0,0 +1,14 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)